Vagrantfile

```diff
@@ -168,16 +173,55 @@ Vagrant.configure('2') do |config|
     end
   end
 
-  config.vm.define :c1_search do |cfg|
+  (1..2).each do |n|
+    symbol = "c1_search#{n}".to_sym
+    config.vm.define symbol do |cfg|
+      default_omnibus config
+      cfg.vm.provider :virtualbox do |vb|
+        vb.customize ['modifyvm', :id, '--memory', 256]
+      end
+      cfg.vm.provision :chef_client do |chef|
+        chef_defaults chef, symbol
+        chef.add_recipe 'cerner_splunk::search_head'
+      end
+      network cfg, symbol
+    end
+  end
+
+  config.vm.define :c1_search3 do |cfg|
+    default_omnibus config
+    cfg.vm.provider :virtualbox do |vb|
+      vb.customize ['modifyvm', :id, '--memory', 256]
+    end
+    cfg.vm.provision :chef_client do |chef|
+      chef_defaults chef, :c1_search3
+      chef.add_role 'sh_captain'
```

Comment:
minor, rather than copying the entire block for 3 and 4, you could make a case statement around line 185 to flex the recipe or role to apply to the node.